templates/github-dev-trends/Readme.md

```
@@ -42,55 +47,77 @@ This is a pnpm monorepo with the following structure:
 ### Installation
 
 If you haven't already, install the Moose CLI and pnpm:
-```bash copy
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)